analysis/quantifier-comparisons/README.md

```diff
@@ -0,0 +1,24 @@
+
+## Docker setup
+
+Running the quantification comparison notebooks are most easily done via Docker, using the `ghcr.io/alexslemonade/scpca-r` image, which contains and RStudio server and all necessary dependencies, including the AWS command line tools and R packages.  AWS credentials are expected to be stored in `~/.aws` on the user's system, having been set up with `aws configure` on the host. https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config
```

Comment:
```suggestion
Running the quantification comparison notebooks are most easily done via Docker, using the `ghcr.io/alexslemonade/scpca-r` image, which contains and RStudio server and all necessary dependencies, including the AWS command line tools and R packages.  
AWS credentials are expected to be stored in `~/.aws` on the user's system, having been set up with `aws configure` on the host. 
https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config
```

Reply:
Only noticed this because I wanted to comment on the first sentence. Is the current state of things that one needs to be logged into GH and a member of Alex's Lemonade to pull the Docker image? Thought that this was an AWS ECR issue.

Reply:
Just noticed the `images/cellranger` README so I think I know the answer to this question now...

Reply:
The `ghcr.io/alexslemonade/scpca-r` image is public, so you should not need to be logged into GH. It is really only there because of dockerhub permissions/pricing. The cellranger image is private (and on AWS ECR), because I didn't want to violate the license.